pkg/controller/defaults/testdata/defaults/volume-class-two-cluster-defaults/expected.yaml

```diff
@@ -0,0 +1,28 @@
+kind: AppInstance
+apiVersion: internal.acorn.io/v1
+metadata:
+  name: app-name
+  namespace: app-namespace
+  uid: 1234567890abcdef
+spec:
+  image: test
+status:
+  observedGeneration: 1
+  namespace: app-created-namespace
+  appImage:
+    id: test
+  appSpec:
+    containers:
+      container-name:
+        image: "image-name"
+        dirs:
+          "/var/tmp":
+            volume: foo
+    volumes:
+      foo: {}
+  conditions:
+    - error: true
+      message: 'cannot establish defaults because two defaults volume classes exist: test-volume-class and test-volume-class-1'
```

Comment:
Will this be flakey? Since the order can't be guaranteed when listing this could be flipped which I observed locally.